plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroup.java

```diff
@@ -104,87 +127,42 @@ private static String getLibraryName(Library library) {
     }
   }
 
-  private void setupDatabinding() {
-    bindingContext = new DataBindingContext(getDisplayRealm());
-    for (Button libraryButton : libraryButtons) {
-      setupDatabindingForButton(libraryButton);
-    }
+  public void dispose() {
```

Comment:
It's called from locations where it has been embedded, like [`MavenAppEngineStandardWizardPage#dispose()`](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java#L303)